src/libraries/System.IO.Compression.ZipFile/src/System.IO.Compression.ZipFile.csproj

```diff
@@ -14,10 +14,24 @@
     <Compile Include="$(CommonPath)System\IO\PathInternal.CaseSensitivity.cs"
              Link="Common\System\IO\PathInternal.CaseSensitivity.cs" />
   </ItemGroup>
+  <!-- Unix specific files -->
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true' ">
```

Comment:
If we're including all of these files for both TargetsUnix and TargetsBrowser, why do we need both the Unix and the Browser target framework configurations?  Is there some ifdef'ing elsewhere that ends up demanding this?  Otherwise, I'd have expected the Unix build to work for browser as well.

Reply:
That was my initial thinking as well. But I guess Browser doesn't "inherit" from Unix, like I thought it did:

https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/eng/targetframeworksuffix.props#L103-L108

Notice that doesn't set `TargetsUnix`, like the rest of them do - Android, OSX, etc.

I was getting build failures on WASM when I just had `Windows` and `Unix` targets:

```
Assembly 'netstandard' is missing dependency 'System.IO.Compression.ZipFile'
```

Adding the Browser target fixed those errors. I noticed this is the same approach System.IO.Compression takes already, so I assume it is the "right" approach:

https://github.com/dotnet/runtime/blob/dfd618dc648ba9b11dd0f8034f78113d69f223cd/src/libraries/System.IO.Compression/src/System.IO.Compression.csproj#L4

https://github.com/dotnet/runtime/blob/dfd618dc648ba9b11dd0f8034f78113d69f223cd/src/libraries/System.IO.Compression/src/System.IO.Compression.csproj#L49-L50

Reply:
Hmm, ok, apparently this changed a year ago:
https://github.com/dotnet/runtime/pull/37944

Reply:
It was news to me 😄